python/chronos/dev/example/run-example-tests-pip-ray.sh

```diff
@@ -83,12 +83,21 @@ sed -i 's/epochs=10/epochs=1/' "${BIGDL_ROOT}/python/chronos/example/quantizatio
 execute_ray_test quantization_tcnforecaster_nyc_taxi "${BIGDL_ROOT}/python/chronos/example/quantization/quantization_tcnforecaster_nyc_taxi.py"
 time7=$?
 
+if [ ! -f ~/.chronos/dataset/nyc_taxi/nyc_taxi_data.csv ]; then
+  wget -nv $FTP_URI/analytics-zoo-data/apps/nyc-taxi/nyc_taxi.csv -P ~/.chronos/dataset/nyc_taxi/
+  mv ~/.chronos/dataset/nyc_taxi/nyc_taxi.csv ~/.chronos/dataset/nyc_taxi/nyc_taxi_data.csv
+fi
+
+execute_ray_test sparkdf_training_nyc_taxi.py "${BIGDL_ROOT}/python/chronos/example/distributed/sparkdf_training_nyc_taxi.py"
```

Comment:
I think you need to set `--datadir ~/.chronos/dataset/nyc_taxi/nyc_taxi_data.csv`

Reply:
ok